Tasks/NuGetCommand/Tests/RestoreTests/multiplesln.ts

```diff
@@ -1,13 +1,14 @@
 import ma = require('vsts-task-lib/mock-answer');
```

Comment:
NuGetRestore was renamed to NuGetCommand -- you need to update make-options.js to change "NuGetRestore" to "NuGetCommand", otherwise "gulp build" or "npm run build" will fail.